backend/app/crud/training.py

```
@@ -0,0 +1,108 @@
+    dates_result = await db.execute(dates_query)
+    performed_dates = dates_result.scalars().all()
+
+    streak_days = 0
+    if performed_dates:
+        # Check if performed today or yesterday to keep streak alive
+        last_date = performed_dates[0]
+        if last_date == today or last_date == today - timedelta(days=1):
+            current_check = today
+            # If not performed today, start checking from yesterday for streak
+            if last_date != today:
+                current_check = today - timedelta(days=1)
```

Ruff (0.14.8)

Local variable `current_check` is assigned to but never used

Remove assignment to unused variable `current_check`

(F841)